hail/python/hail/methods/impex.py

```diff
@@ -2399,8 +2397,6 @@ def import_vcf(path,
            _external_header=nullable(str))
 def import_gvcfs(path,
                  partitions,
-                 force=False,
-                 force_bgz=False,
```

Comment:
Should update the docstring about "the only difference" with the args of `import_vcf` to mention the lack of `force, force_bgz`.